config/owasp/suppressions.xml

```diff
@@ -44,4 +44,11 @@
                 ]]></notes>
         <cve>CVE-2020-23171</cve>
     </suppress>
+    <suppress until="2030-01-01">
+        <notes><![CDATA[
+                 https://nvd.nist.gov/vuln/detail/CVE-2021-22096#range-7096083
+                 https://cwe.mitre.org/data/definitions/117.html
```

Comment:
@sabahirfan can you replace with the following comment `Temporary suppression until upgrade to Spring Boot 2.4.x`

Reply:
done